appconfig/process.csv

```diff
@@ -1,21 +1,21 @@
 host,port,proctype,procname,U,localtime,g,T,w,load,startwithall,extras,qcmd
-localhost,{KDBBASEPORT}+1,discovery,discovery1,appconfig/passwords/accesslist.txt,1,0,,,${KDBCODE}/processes/discovery.q,1,,q
-localhost,{KDBBASEPORT},tickerplant,tickerplant1,appconfig/passwords/accesslist.txt,1,0,,,${KDBCODE}/processes/tickerplant.q,1,-schemafile database -tplogdir ${TORQHOME}/hdb,q
-localhost,{KDBBASEPORT}+2,rdb,rdb1,appconfig/passwords/accesslist.txt,1,1,3,,${KDBCODE}/processes/rdb.q,1,,q
-localhost,{KDBBASEPORT}+3,hdb,hdb1,appconfig/passwords/accesslist.txt,1,1,60,4000,${KDBHDB},1,,q
-localhost,{KDBBASEPORT}+4,hdb,hdb2,appconfig/passwords/accesslist.txt,1,1,60,4000,${KDBHDB},1,,q
-localhost,{KDBBASEPORT}+5,wdb,wdb1,appconfig/passwords/accesslist.txt,1,1,,,${KDBCODE}/processes/wdb.q,1,,q
-localhost,{KDBBASEPORT}+6,sort,sort1,appconfig/passwords/accesslist.txt,1,1,,,${KDBCODE}/processes/wdb.q,1,-s -2,q
-localhost,{KDBBASEPORT}+7,gateway,gateway1,appconfig/passwords/accesslist.txt,1,1,,4000,${KDBCODE}/processes/gateway.q,1,,q
+localhost,{KDBBASEPORT}+1,discovery,discovery1,${BASEDIR}/appconfig/passwords/accesslist.txt,1,0,,,${KDBCODE}/processes/discovery.q,1,,q
```

Comment:
why is this change needed? why is it using an env var called `BASEDIR` and not `TORQHOME` or something?